projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -137,6 +127,40 @@ public function register_rest_routes() {
 				),
 			)
 		);
+
+		// Create a Jetpack Social connection.
+		register_rest_route(
+			'jetpack/v4',
+			'/social/connections/new',
+			array(
+				'methods'             => WP_REST_Server::CREATABLE,
+				'callback'            => array( $this, 'create_publicize_connection' ),
+				'permission_callback' => array( $this, 'require_author_privilege_callback' ),
+				'schema'              => array( $this, 'get_jetpack_social_connections_schema' ),
+			)
+		);
+
+		// Update a Jetpack Social connection.
+		register_rest_route(
+			'jetpack/v4',
+			'/social/connections/(?P<connection_id>\d+)',
+			array(
+				'methods'             => WP_REST_Server::CREATABLE,
```

Comment:
We could use [EDITABLE](https://github.com/WordPress/wordpress-develop/blob/6.5/src/wp-includes/rest-api/class-wp-rest-server.php#L40) here so tha this works for PUT requests etc.